ddk-configuration/.fbprefs-test

```diff
@@ -0,0 +1,182 @@
+#FindBugs User Preferences

```

Comment:
Is this really necessary? The problem with this is that Findbugs cannot use the same config for the Maven and Eclipse plug-in. So what you are doing here will only affect the Eclipse plug-in while the Maven plug-in just uses the config from the parent pom. The settings we had so far was more or less consistent. If we also want to have different FIndbugs settings for normal and test code we have to do a bit more to get this consistent.

Reply:
I think this was then re-deleted in some later commit.  Maybe I should have squashed the 2, but I didn't separate the commits as well as I'd wanted to.

Reply:
Nope, this file is finally in (I'm looking at changes from all commits). I think we should either go for separate settings (normal vs. tests) for all SCA tools or for none. In this pull request we remove test settings for Checkstyle and PMD but add them for Findbugs (Eclipse only). OK for you to remove the .fbprefs-test (incl. references from .project files)?

Reply:
Ok, removed in latest commit. As well as the project links